pysolarmanv5/pysolarmanv5.py

```diff
@@ -299,18 +320,28 @@ def _send_receive_v5_frame(self, data_logging_stick_frame):
         return v5_response
 
     def _received_frame_is_valid(self, frame):
-        """Check that the frame is valid and that the serial number of the received
+        """
+        Check that the frame is valid and that the serial number of the received
         frame matches with the last sent one.
-        Ignore also any frames with control code 0x4710 (counter frame).
         """
         if not frame.startswith(self.v5_start):
             self.log.debug("[%s] V5_MISMATCH: %s", self.serial, frame.hex(" "))
             return False
         if frame[5] != self.sequence_number:
             self.log.debug("[%s] V5_SEQ_NO_MISMATCH: %s", self.serial, frame.hex(" "))
             return False
+        return True
+
+    def _handle_protocol_frame(self, frame):
```

Comment:
I did choose rather generic name cause it could be place for handling broader range of protocol packets in the future.